trigger/patch.go

```diff
@@ -273,15 +273,15 @@ func (t *patchTriggers) makeData(sub *event.Subscription) (*commonTemplateData,
 			Color:     slackColor,
 		})
 	}
-	var makespan time.Duration
-	if utility.IsZeroTime(t.patch.FinishTime) {
-		patchTasks, err := task.Find(task.ByVersion(t.patch.Id.Hex()))
-		if err == nil {
-			_, makespan = task.GetTimeSpent(patchTasks)
-		}
-	} else {
-		makespan = t.patch.FinishTime.Sub(t.patch.StartTime)
+
+	tasks, err := task.Find(task.ByVersion(t.patch.Id.Hex()))
```

Comment:
If the issue is that we do this where there's no start or end time, should we just now only do this query if one of finished / started is zero?

Reply:
I may be misunderstanding the question. There was a formatting issue for containers tasks that failed to start after several attempts and were missing a start time or an end time for tasks. We need the list of tasks for the patch regardless.

Reply:
Ah I was referencing: 
> This standardizes how the time taken is formatted to avoid a very large integer in notifications for cases where there is no start time or an end time.

do we need the list of tasks only because of container tasks? Is there a way we can roll this up so we can avoid needing to do this?

Reply:
We need the list of tasks for all cases so that we can iterate over it and look for the task with the earliest start time and latest finish time and use it to calculate the time taken. It's not something that was introduced for container tasks.

Reply:
What it was doing, where it didn't calculate the time spent if the finishTime wasn't ZeroTime wasn't a good approach, it's actually what caused it to have a huge number in the notification. You can see that looking at the data for the patch (attached below). Instead, it should always be calculated based on the tasks earliest start time and latest finish time, which is what spruce does when calculating Makespan. 
![image](https://github.com/evergreen-ci/evergreen/assets/13104717/e2b9f2fe-bd2c-4b70-af87-be58695ca703)

Reply:
Makes sense, thanks!